frontend/src/components/ActionLink/ActionLinkwithClick.js

```diff
@@ -68,6 +70,7 @@ function ActionLinkwithClick({
           timestamp: new Date(),
         });
         remove && markThreadAsDeleted(threadId) && markThreadAsInactive();
+        onReplyPress();
```

Comment:
You will need an if statement here to check if there is an `onReplyPress` function` as the behaviour is a bit weird for the other buttons that aren't reply.
```suggestion
        if (ononReplyPress) onReplyPress();
```

Reply:
Done.